listenbrainz/webserver/templates/navbar.html

```diff
@@ -72,21 +73,32 @@
               {{ current_user.musicbrainz_id }} <span class="caret"></span>
             </a>
             <ul class="dropdown-menu" role="menu">
-              <li><a href="{{ url_for('user.profile', user_name=current_user.musicbrainz_id) }}">My Listens</a></li>
-              <li><a href="{{ url_for('index.feed') }}">My Feed</a></li>
-              <li><a href="{{ url_for('user.playlists', user_name=current_user.musicbrainz_id) }}">My Playlists</a></li>
```

Comment:
"My Listens" and "My Feed" are duplicated in the nav bar, but there is no way to access "My playlists" directly so I think it should remain there.

As I commented on the PR already I think the recommendation below should also remain here until we design a better music discovery page

Reply:
Makes sense. I will revert the `For me` changes and let the tracks discovery stay for now as we set some time to think of how the implementation can be done in a better way.